randovania/games/prime1/json_data/Tallon Overworld.txt

```diff
@@ -1282,7 +1282,12 @@ Extra - aabb: [-123.91175, 156.58524, -93.03604, -94.827385, 177.84967, -13.4457
                   Morph Ball Bomb and Morph Ball and Bomb Jump (Advanced)
           All of the following:
               # Slowly Float Up Method
-              Space Jump Boots and Slope Jump (Advanced) and Gravityless Underwater Movement (Expert)
+              Slope Jump (Advanced) and Gravityless Underwater Movement (Expert)
+              Any of the following:
+                  Space Jump Boots
+                  All of the following:
+                      # https://www.youtube.com/watch?v=c4NX1k6p-gE
+                      Morph Ball Bomb and Morph Ball and Bomb Jump (Advanced)
```

Comment:
The NSJ slope jump should be expert in place of the advanced with SJ

Reply:
Do you mean the slope jump is advanced with SJ and expert NSJ, or expert in general?